v3/pkg/application/linux_cgo.go

```diff
@@ -1010,17 +1010,41 @@ func onDragNDrop(target unsafe.Pointer, context *C.GdkDragContext, x C.gint, y C
 
 //export onKeyPressEvent
 func onKeyPressEvent(widget *C.GtkWidget, event *C.GdkEventKey, userData unsafe.Pointer) C.gboolean {
-	windowId := uint(*((*C.uint)(userData)))
-	fmt.Println("onKeyPressEvent", windowId)
-	/*
-		windowKeyEvents <- &windowKeyEvent{
+    accelerator := getKeyboardState(event)
+	windowKeyEvents <- &windowKeyEvent{
 			windowId:          windowID,
```

Comment:
`windowID` is not defined any longer.  Please grab line 1013 back in and use it.